### DISCUSSION TITLE
图数据库学习路径 - siwei.io

### DISCUSSION THREAD

**Discussion by giscus** [siwei-io/siwei-io.github.io#44] [Comments]
# 图数据库学习路径 - siwei.io

 您可以通过这个导图开始了解、入门图数据库和 Nebula Graph，上边的区域是入门的部分，下边还有进阶的部分，我们会持续更新、维护这个导图的~
 Tips:
 加载成功之后，点击 [See the board] 可以开始浏览，双击放大缩小。 或者点导图区域右上角的箭头在新的窗口打开，可以更方面浏览器中内容。   

https://www.siwei.io/path/

**Comment by amour7714** [comment]
感谢！
